ferrocene/doc/evaluation-report/src/rustfmt/tool-analysis.rst

```diff
@@ -0,0 +1,72 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. default-domain:: qualification
+
+Tool Analysis
+=============
+
+Potential Errors
+----------------
+
+.. list-table::
+   :align: left
+   :header-rows: 1
+   :widths: 15, 15, 25, 20, 25
+
+   * - Error identifier
+     - Use case
+     - Description
+     - Risk
+     - Mitigation
+   * - .. id:: RUSTFMT_ERR_01
+     - :id:`RUSTFMT_UC1_FMT`
+     - rustfmt emits code with changed semantics
+     - Undefined behavior
```

Comment:
The risk is not just UB, but also different semantic meaning of the code.

Reply:
In my head changing the semantics will produce undefined behaviour, isn't it? Because the behaviour will be different somehow, but we cannot define what it will do different.

What do you think about "Undefined behavior, due to changed semantics"?

Reply:
I would not use the "undefined behavior" wording here, because it might be confused with the compiler's meaning of undefined behavior. I would probably have:

* "rustfmt emits code with a different semantic meaning than the input" as the description
* "Change in program semantics" as the risk.